Kconfig.nrf

```diff
@@ -113,6 +113,12 @@ config FLASH_SIZE
 	default 1524
 endif
 
+config NRF_SECURITY_ENABLER
+	def_bool y
+	imply NRF_SECURITY if BT_CRYPTO || BT_HOST_CRYPTO || BT_MESH_USES_MBEDTLS_PSA || BT_FAST_PAIR_CRYPTO_PSA
+	imply ENTROPY_GENERATOR if BOARD_NATIVE_SIM
+	imply PSA_WANT_GENERATE_RANDOM if BT_HOST_CRYPTO_PRNG
+	depends on !SOC_NRF5340_CPUNET
```

Comment:
Why was this added? This breaks things when using BT host on 53net

Reply:
Currently the CPUNET (and controller code) isn't PSA crypto ready. It will only set nrf_oberon library and use proprietary APIs (`ocrypto_` prefixed)